pkgs/applications/networking/localsend/default.nix

```diff
@@ -0,0 +1,64 @@
+{ lib, stdenv, appimageTools, fetchurl, undmg }:
+
+let
+  pname = "localsend";
+  version = "1.7.0";
+
+  srcs = {
+    x86_64-linux = fetchurl {
+      url = "https://github.com/localsend/localsend/releases/download/v${version}/LocalSend-${version}.AppImage";
+      hash = "sha256-C0mHKIdvh0PjiP+lTPiBu8M675WywhIgRwBjGKqWOLw=";
+    };
+    x86_64-darwin = fetchurl {
+      url = "https://github.com/localsend/localsend/releases/download/v${version}/LocalSend-${version}.dmg";
+      hash = "sha256-B6t8T7MSUOD6nTzjEA7TX/IdK5RM+GORVDfrS4HSAQw=";
+    };
+  };
+  src = srcs.${stdenv.hostPlatform.system};
```

Comment:
This is missing an or throw unsupported system otherwise it will create a syntax error on other platforms.

Reply:
Added.